contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -2515,13 +2679,22 @@ read_param_def(InlineCodeBlockArgs *args, const char *paramdefstr)
 		return;
 	}
 
+	/* 
+	 * Put delimiters around parameter names like @@par or @par#.
+	 * Reserving twice the amount of space: since the shortest possible parameter
+	 * definition is 5 characters ('@@p x' , where x would be the type), this will 
+	 * always be enough for adding delimiters.
+	 */	
+	paramdefstr_delimited = (char *) palloc(sizeof(char *)*strlen(paramdefstr)*2);	
```

Comment:
`sizeof(char *)` is probably not correct here - I think you probably meant just `sizeof(char)`. Also, why is it not possible to just allocate `strlen(paramdefstr)+1+(2*<size_of_delimiter_string>)`?

Reply:
Scratch that, I didn't realize that multiple parameters can be contained within this string. My previous point about using `sizeof(char)` still stands though

Reply:
On 1: you;re right, will fix.
On 2: There can be N parameters in the string that need to be delimited so that's N*2 additional chars. We should not make assumptions about the value of N, so this seems the formula that is safest + simplest.

Reply:
fixed